src/dbnode/storage/namespace.go

```diff
@@ -293,8 +296,8 @@ func newDatabaseNamespace(
 
 	scope := iops.MetricsScope().SubScope("database").
 		Tagged(map[string]string{
-			"namespace": id.String(),
-		})
+		"namespace": id.String(),
```

Comment:
Maybe github is messing up, but deleting this whitespace seems wrong? Should this not be indented

Reply:
@Haijuncao bump